### PR TITLE
[Frontend] Fix decomposition tests

### PR DESCRIPTION
**Pull request by erick-xanadu**
**Context:**  Recent changes in the TOML files broke some tests.

**Description of the Change:** Fix those tests.

**Benefits:** Latest/latest/latest passes again (at least for lightning.qubit).

Comment:

The minimum version of pennylane-lightning required is 0.37.0-dev7 but those wheels failed to be uploaded. I have used 0.37.0-dev9. If the minimum version required is required for this PR, let me know and I will ask the team to push wheels for -dev7

[sc-63376]

### CODE REVIEW

File: doc/changelog.md [hunk 1]
```diff
@@ -132,6 +132,9 @@
         (array([0.5, 0. , 0.5, 0. ]),)  
   ```
   
+* Fix decomposition tests.
+  [(#745)](https://github.com/PennyLaneAI/catalyst/pull/745)
```
Comment (dime10) [inline doc/changelog.md:136]: I don't think this PR contains any changes to the package, as such it probably shouldn't feature a changelog entry.

File: doc/changelog.md [hunk 2]
```diff
@@ -132,6 +132,9 @@
...
+
```
Comment (erick-xanadu) [inline doc/changelog.md:137]:
```suggestion
```

File: doc/changelog.md (hunk 1 above)
Comment (erick-xanadu) [reply doc/changelog.md]: See here: https://github.com/PennyLaneAI/catalyst/pull/745/commits/c1297dd974885d5ca460e615c1e2d25de3ecddd9

**Comment by dime10** [comment]
I'm not sure how much of this PR is clashing with #718 and #712, could you double check?

**Comment by erick-xanadu** [comment]
> I'm not sure how much of this PR is clashing with https://github.com/PennyLaneAI/catalyst/pull/718 and https://github.com/PennyLaneAI/catalyst/pull/712, could you double check?

I'm happy to close this PR without merging if #718 and #712 would fix latest/latest/latest.

**Comment by grwlf** [comment]
> > I'm not sure how much of this PR is clashing with #718 and #712, could you double check?
> 
> I'm happy to close this PR without merging if #718 and #712 would fix latest/latest/latest.

We need #712 . I would suggest to merge it instead of this PR. #712 addresses exactly this issue, but in my opinion - in a more systematic way: it adds an option to modify capabilities directly so we no longer need to stick to a particular toml syntax in tests.

The #718 fixes another toml leftover, it requires #712 as a prerequisite.

**Comment by dime10** [comment]
> > > I'm not sure how much of this PR is clashing with #718 and #712, could you double check?
> > 
> > 
> > I'm happy to close this PR without merging if #718 and #712 would fix latest/latest/latest.
> 
> We need #712 . I would suggest to merge it instead of this PR. #712 addresses exactly this issue, but in my opinion - in a more systematic way: it adds an option to modify capabilities directly so we no longer need to stick to a particular toml syntax in tests.
> 
> The #718 fixes another toml leftover, it requires #712 as a prerequisite.

Let's give those PRs the final push! @grwlf can you ensure relevant discussions are resolved with the reviewers and merged this week?

**Comment by erick-xanadu** [comment]
@dime10 and @grwlf just commenting that this is probably no longer required, right?

**Comment by dime10** [comment]
> @dime10 and @grwlf just commenting that this is probably no longer required, right?

Probably not 👍

**Pull request closed without merging**